pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -800,6 +809,35 @@ func (p PITRSpec) Disabled() PITRSpec {
 	return p
 }
 
+type BackupTimeouts struct {
+	Starting *uint32 `json:"startingStatus,omitempty"`
+}
+
+type BackupOptions struct {
+	OplogSpanMin float64           `json:"oplogSpanMin"`
+	Priority     map[string]string `json:"priority,omitempty"`
+	Timeouts     *BackupTimeouts   `json:"timeouts,omitempty"`
+
+	// TODO: Do we need to set these here since we have them in BackupTaskSpec?
+	// Compression      compress.CompressionType `json:"compression,omitempty"`
+	// CompressionLevel *int                     `json:"compressionLevel,omitempty"`
```

Comment:
We set `Compression` and `CompressionLevel` in `BackupTaskSpec`, I think we don't need to have these options here as well, wdyt @egegunes @hors

Reply:
I agree, we have them for scheduled backups and on demand backups alread